apps/gk_neut_species_react.c

```diff
@@ -72,60 +85,99 @@ gk_neut_species_react_cross_init(struct gkyl_gyrokinetic_app *app, struct gk_neu
         .charge_state = react->react_type[i].charge_state, 
         .type_self = react->type_self[i], 
         .all_gk = false, 
-        .base = 0,
       };
       react->recomb[i] = gkyl_dg_recomb_new(&recomb_inp, app->use_gpu);       
-    } 
+    }
+    else if (react->react_id[i] == GKYL_REACT_CX) {
+      struct gkyl_dg_cx_inp cx_inp = {
+        .grid = &s->grid,
+	.cbasis = &app->confBasis,
+	.pbasis_gk = &app->basis,
+	.pbasis_vl = &app->neut_basis,
+	.conf_rng = &app->local,
+	.conf_rng_ext = &app->local_ext,
+	.phase_rng = &s->local,
+	.mass_ion = react->react_type[i].ion_mass,
+	.mass_neut = react->react_type[i].partner_mass,
+	.type_ion = GKYL_ION_D,
```

Comment:
Should this really always be GKYL_ION_D? Or should it be based on the input file since now we are supporting neon, etc

Reply:
Fixed in latest commit.